backend/internal/litellm/client_test.go

```
@@ -0,0 +1,168 @@
+		{"alpha", "http://llamarack/v1", "sk-two"},
+	}
+	for _, tc := range cases {
+		if !entryDrifted(entry, tc.instanceID, tc.apiBase, tc.apiKey) {
+			t.Fatalf("expected drift for %+v", tc)
+		}
+	}
+	entry.ModelInfo.LlamaRackInstanceID = "stale"
+	if !entryDrifted(entry, "alpha", "http://llamarack/v1", "sk-one") {
+		t.Fatal("expected instance id drift")
+	}
+	if entryDrifted(entry, "alpha", "http://llamarack/v1", "sk-one") && entry.ModelName == "alpha" && entry.LiteLLMParams.Model == "openai/alpha" {
```

golangci-lint (2.13.2)

[error] SA9003: empty branch

(staticcheck)